soh/soh/UIWidgets.cpp

```diff
@@ -440,15 +438,14 @@ namespace UIWidgets {
             ImGui::SameLine();
             ImGui::SetCursorPosX(ImGui::GetCursorPosX() - 7.0f);
         }
-        if (PlusMinusButton) {
+
         #ifdef __SWITCH__
-            ImGui::PushItemWidth(ImGui::GetWindowSize().x - 110.0f);
+            ImGui::PushItemWidth(ImGui::GetContentRegionAvail().x - 110.0f);
         #elif defined(__WIIU__)
-            ImGui::PushItemWidth(ImGui::GetWindowSize().x - 79.0f * 2);
+            ImGui::PushItemWidth(ImGui::GetContentRegionAvail().x - 79.0f * 2);
         #else
-            ImGui::PushItemWidth(ImGui::GetWindowSize().x - 79.0f);
+            ImGui::PushItemWidth(ImGui::GetContentRegionAvail().x - 79.0f);
```

Comment:
want to verify this works as intended on all platforms, getting someone to test on switch/wii u (or emulator equivalent) would be great!

Reply:
I think these calls to set the width are completely unnecessary at this point, so my next commit will remove them, but yes, I will definitely be asking for verification on those platforms!